felix/dataplane/linux/int_dataplane.go

```diff
@@ -426,7 +426,8 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 	ConfigureDefaultMTUs(hostMTU, &config)
 	podMTU := determinePodMTU(config)
 	if err := writeMTUFile(podMTU); err != nil {
-		log.WithError(err).Error("Failed to write MTU file, pod MTU may not be properly set")
+		log.WithError(err).Error("Failed to write MTU file shutting down")
+		return nil
```

Comment:
@caseydavenport @tomastigera would you know the implications of running with no MTU file vs. shutting down? We'll be at risk of introducing a new crashloop here, but what was the problem before?